t/integration/tasks.py

```diff
@@ -41,3 +41,20 @@ def cumadd(requests: List[SimpleRequest]) -> None:
     for request in requests:
         result += request.args[0]
         current_app.backend.mark_as_done(request.id, result, request=request)
+
+
+@shared_task(base=Batches, flush_every=2, flush_interval=1)
```

Comment:
I made this much shorter in #60, so let's match that.

```suggestion
@shared_task(base=Batches, flush_every=2, flush_interval=0.1)
```